### PR TITLE
homework functions

### PR DESCRIPTION
**Pull request by sudakevych-andrii**

### CODE REVIEW

File: git/homework.py [hunk 1]
```diff
@@ -11,132 +11,64 @@ class OurAwesomeException(Exception):
...
-
-    Examples:
-        is_word_in_text("Hello", "Hello word")
-        >>> True
-        is_word_in_text("Glad", "Nice to meet you ")
-        >>> False
-
-    """
-    pass
+    return text.find(word) != -1
```
Comment (alchemixXx) [inline git/homework.py]: could use just 'return text in word'

File: git/homework.py [hunk 2]
```diff
@@ -11,132 +11,60 @@ class OurAwesomeException(Exception):
 
 
 def is_two_object_has_same_value(first: Any, second: Any) -> bool:
-    """
-    If @first and @second has same value should return True
-    In another case should return False
-    """
-    pass
+    return True if first == second else False
```
Comment (dimigor) [inline git/homework.py]: You do not need ternary operator here, Operand `==` returns bool

File: git/homework.py [hunk 3]
```diff
@@ -11,132 +11,60 @@ class OurAwesomeException(Exception):
...
 
 
 def is_two_objects_has_same_type(first: Any, second: Any) -> bool:
-    """
-    If @first and @second has same type should return True
-    In another case should return False
-    """
-    pass
+    return True if type(first) == type(second) else False
```
Comment (dimigor) [inline git/homework.py]: Same as above

File: git/homework.py [hunk 4]
```diff
@@ -11,132 +11,60 @@ class OurAwesomeException(Exception):
...
 
 
 def is_two_objects_is_the_same_objects(first: Any, second: Any) -> bool:
-    """
-    If @first and @second has same type should return True
-    In another case should return False
-    """
-    pass
+    return True if first is second else False
```
Comment (dimigor) [inline git/homework.py]: `is` also return bool

File: git/homework.py [hunk 5]
```diff
@@ -11,132 +11,60 @@ class OurAwesomeException(Exception):
...
-    """
-    Create dict which keys is alphabetic characters. And values their number in alphabet
-    Notes You could see an implementaion of this one in test, but create another one
-    Examples:
-        alphabet()
-        >>> {"a": 1, "b": 2 ...}
-    """
-    pass
+    a = ord('a')
+    alphabetDict = {}
```
Comment (dimigor) [inline git/homework.py]: name your variables using PEP8 conversation. In your case, it will be `alphabet_dict`

File: git/homework.py [hunk 6]
```diff
@@ -11,132 +11,60 @@ class OurAwesomeException(Exception):
...
+    index = 1
+    for i in range(a, a + 26):
```
Comment (dimigor) [inline git/homework.py]: use alphabet python library instead

File: git/tests.py [hunk 7]
```diff
@@ -163,3 +163,7 @@ def test_alphabet(self):
 
 if __name__ == "__main__":
     unittest.main()
+
+
+print("Done!")
```
Comment (dimigor) [inline git/tests.py]: Please, do not comment prints:)

File: git/homework.py [hunk 8]
```diff
@@ -4,139 +4,62 @@
...
 def alphabet() -> dict:
-    """
-    Create dict which keys is alphabetic characters. And values their number in alphabet
-    Notes You could see an implementaion of this one in test, but create another one
-    Examples:
-        alphabet()
-        >>> {"a": 1, "b": 2 ...}
-    """
-    pass
+    return dict(zip(ascii_lowercase, range(1, 27)))
```
Comment (yuriimchg) [inline git/homework.py]: на цьому завданні падає тест; 
тут варто поміняти місцями символи і числа

File: git/homework.py (hunk 1 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/homework.py (hunk 2 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/homework.py (hunk 3 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/homework.py (hunk 4 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/homework.py (hunk 5 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/homework.py (hunk 6 above)
Comment (sudakevych-andrii) [reply git/homework.py]: Done

File: git/tests.py (hunk 7 above)
Comment (sudakevych-andrii) [reply git/tests.py]: Done

**Pull request merged**